applications/StructuralMechanicsApplication/custom_processes/impose_rigid_movement_process.cpp

```diff
@@ -10,13 +10,16 @@
 //
 
 // System includes
+#ifdef _OPENMP
+#include <omp.h>
+#else
+#include <ctime>
```

Comment:
why do you need this?

Reply:
actually openmp_utils is designed for you not to import omp.h...